x-pack/plugins/synthetics/server/routes/settings/enable_default_alerting.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { FindActionResult } from '@kbn/actions-plugin/server';
+import { UptimeServerSetup } from '../../legacy_uptime/lib/adapters';
+import { savedObjectsAdapter } from '../../legacy_uptime/lib/saved_objects';
+import { populateAlertActions, RuleAction } from '../../../common/rules/alert_actions';
+import {
+  ACTION_GROUP_DEFINITIONS,
+  SYNTHETICS_ALERT_RULE_TYPES,
+} from '../../../common/constants/synthetics_alerts';
+import { SyntheticsRestApiRouteFactory } from '../../legacy_uptime/routes/types';
+import { SYNTHETICS_API_URLS } from '../../../common/constants';
+import { UptimeRequestHandlerContext } from '../../types';
+import { SyntheticsMonitorStatusTranslations } from '../../../common/rules/synthetics/translations';
+
+export const enableDefaultAlertingRoute: SyntheticsRestApiRouteFactory = () => ({
+  method: 'POST',
+  path: SYNTHETICS_API_URLS.ENABLE_DEFAULT_ALERTING,
+  validate: {
+    body: schema.object({
+      isDisabled: schema.maybe(schema.boolean()),
+      isUpdated: schema.maybe(schema.boolean()),
+    }),
+  },
+  writeAccess: true,
+  handler: async ({ context, server, savedObjectsClient, request }): Promise<any> => {
+    const { isUpdated } = (request.body ?? {}) as { isDisabled?: string; isUpdated?: boolean };
+
+    return createDefaultAlertIfNotExist({
+      context,
+      isUpdated,
+      soClient: savedObjectsClient,
+      server,
+    });
+  },
+});
+
+export const getDefaultAlertingRoute: SyntheticsRestApiRouteFactory = () => ({
+  method: 'GET',
+  path: SYNTHETICS_API_URLS.ENABLE_DEFAULT_ALERTING,
+  validate: {},
+  handler: async ({ context }): Promise<any> => {
+    const rulesClient = (await context.alerting)?.getRulesClient();
+
+    const { data } = await rulesClient.find({
+      options: {
+        page: 1,
+        perPage: 1000,
```

Comment:
Since we only have 1 rule, do we need to do this per page. Or is there the ability to add multiple rules that we decided on?